src/client/Microsoft.Identity.Client/Internal/JsonWebTokenConstants.cs

```diff
@@ -11,6 +11,7 @@ internal class JsonWebTokenConstants
         internal class Algorithms
         {
             public const string RsaSha256 = "RS256";
+            public const string EdcSha256 = "ES256";
```

Comment:
This is what I was referring to. Elliptic curve can be ES256/ES384/ES512 etc. This value will need to be updated based on the conical JWK created by the elliptic curve keys.

Reply:
How does this relate to the list of algorithms in https://docs.microsoft.com/en-us/azure/key-vault/keys/about-keys#signverify and https://docs.microsoft.com/en-us/azure/key-vault/keys/about-keys#signverify1 ?